pages/glossary/hovered.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Hovered
+key: hovered
+unambiguous: true
+objective: true
+---
+
+An element is hovered (by a pointing device) when that element matches the [`:hover` pseudo-class](https://drafts.csswg.org/selectors-4/#hover-pseudo).
```

Comment:
This is generic (not specific for HTML), but would probably benefit from adding
"for HTML elements, see also https://html.spec.whatwg.org/multipage/semantics-other.html#selector-hover" which is much more detailed than the CSS version.

Reply:
Updated